src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -797,7 +797,51 @@ private bool IsOpenable(bool needToUncompress, bool needToLoadIntoMemory, out st
             return true;
         }
 
-        private bool SizesTooLarge() => _compressedSize > uint.MaxValue || _uncompressedSize > uint.MaxValue;
+        private bool AreSizesTooLarge => _compressedSize > uint.MaxValue || _uncompressedSize > uint.MaxValue;
+
+        private static CompressionLevel MapCompressionLevel(BitFlagValues generalPurposeBitFlag, CompressionMethodValues compressionMethod)
```

Comment:
These two methods don't appear to be related to this change.

Reply:
thanks for noticing, they're from an unrelated change that was in main (accidentally included when fixing the merge conflict)

Reply:
Fixed